### PR TITLE
Smooth scroll for careers page

### PR DESCRIPTION
**Pull request by zarSou9**

### CODE REVIEW

File: app/routes/how-can-i-help.career.tsx [hunk 1]
```diff
@@ -584,34 +584,43 @@ const FieldBuildingPath = () => (
...
-      />
-    </div>
-  </>
-)
+const CareerPaths = () => {
+  const smoothScroll = (id: string) => {
+    const element = document.getElementById(id)
+    if (element) {
+      element.scrollIntoView({behavior: 'smooth'})
+      window.history.pushState(null, '', `#${id}`)
```
Comment (zarSou9) [inline app/routes/how-can-i-help.career.tsx:592]: @mruwnik I tried with remix's `navigate`, but it just jumped to the section. is `history.pushState` alright?

Comment (mruwnik) [reply app/routes/how-can-i-help.career.tsx:592]: how about just applying `scroll-behavior: smooth;` to the page?

Comment (zarSou9) [reply app/routes/how-can-i-help.career.tsx]: Cus I'm a silly guy, and claude didn't suggest it for some reason

Comment (mruwnik) [reply app/routes/how-can-i-help.career.tsx]: took me years to get to the stage where I always triple check for a css solution :D

**Review by mruwnik — APPROVED** [review]

**Pull request merged**